packages/evm/contracts/test/court/AragonCourtMock.sol

```diff
@@ -34,29 +34,30 @@ contract AragonCourtMock is AragonCourt, TimeHelpersMock {
     {}
 
     function setDisputeManager(address _addr) external {
-        _setModule(DISPUTE_MANAGER, _addr);
+        _setAndCacheModule(DISPUTE_MANAGER, _addr);
     }
 
     function setDisputeManagerMock(address _addr) external {
         // This function allows setting any address as the DisputeManager module
```

Comment:
Yes, the main reason is because this is being used to force a controlled address as the dispute manager.
However, hope it has been clarified in 797f34c148e1085dc338ff724c6eae72649f251e